src/main/kotlin/no/nav/familie/ef/sak/service/steg/IverksettMotOppdragSteg.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.familie.ef.sak.service.steg
+
+import no.nav.familie.ef.sak.repository.domain.Behandling
+import no.nav.familie.ef.sak.service.TilkjentYtelseService
+import no.nav.familie.ef.sak.task.StatusPåOppdragTask
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+
+@Service
+class IverksettMotOppdragSteg(private val tilkjentYtelseService: TilkjentYtelseService,
+                              private val taskRepository: TaskRepository) : BehandlingSteg<Void?> {
+
+    override fun validerSteg(behandling: Behandling) {
+        // TODO
```

Comment:
Todos kan være fint å kommentere, hva som savnes, eller lenke til favrooppgave, slik att man vet att de blir fiksede noen gang. Hvis ikke blir de liggendes for evig

Reply:
Gott poæng! :)